### PR TITLE
Azure Batch Test Script

### PR DESCRIPTION
**Pull request by homezcx**

### CODE REVIEW

File: bin/runazuretests.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+fi
+
+CLASSPATH=$1
+TESTCLASS=$2
+
+CMD="java -cp ${CLASSPATH} org.junit.runner.JUnitCore ${TESTCLASS}"
+
+echo -e "\n\nRunning Azure Batch Tests...\n\n"
+echo $CMD
+$CMD # 2> /dev/null
```
Comment (sharathmalladi) [inline bin/runazuretests.sh]: Why "2> /dev/null"? Would we not want the user to see errors from the executing command?

Comment (homezcx) [reply bin/runazuretests.sh]: I will remove that. Get that from other file.

File: bin/runazuretests.sh [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	echo "Only 2 arguments are accepted - CLASSPATH and TESTCLASS"
+	exit 1;
+fi
+
+[ -z "$VARIABLE" ] && VARIABLE="REEF_TEST_AZBATCH"
+
+if [ $VARIABLE != "true" ]
+then
+    echo "Trying to set REEF_TEST_AZBATCH environment variable."
+	echo "Please run as \". runazuretests.sh\" or set it from your environment."
```
Comment (ryumbra) [inline bin/runazuretests.sh]: indentation seems to be off in this line and the one below.

**Review by ryumbra — APPROVED** [review]

File: bin/runazuretests.sh (hunk 2 above)
Comment (homezcx) [reply bin/runazuretests.sh]: fixed.

**Pull request merged**